megamek/src/megamek/common/verifier/TestBattleArmor.java

```diff
@@ -1177,8 +1177,8 @@ public StringBuffer printEntity() {
         buff.append(printSource());
         buff.append(printShortMovement());
         if (correctWeight(buff, true, true)) {
-            buff.append("Weight: ").append(getWeight()).append(" (")
-                    .append(calculateWeight()).append(")\n");
+            buff.append("Weight: ").append(getWeight() * 1000).append(" kg (")
```

Comment:
Does this need the `%3.1f` format like below?

Reply:
I don't see the format string you're referring to.